app/views/collection/new_mobile_user.html.slim

```diff
@@ -0,0 +1,3 @@
+.litebox-embed(style="width:550px")
```

Comment:
Maybe this set `width` attribute is why the `litebox` looks weird on mobile

Reply:
Can we set the width narrower in this instance?